<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>799</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interface Drone v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Drone controls</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Camera view :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lbl_cam">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>280</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>280</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Connexion to the drone required</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Drone commands :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Informations :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Battery : </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Altitude (m) : </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcd_alt">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="intValue" stdset="0">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Grayscale : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lbl_rendu">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>280</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>280</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Connexion to the drone required</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_6">
              <attribute name="title">
               <string>Anim/Keyboard ctrl</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="btn_anim">
                 <property name="text">
                  <string>Drone animation</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="txt_anim"/>
               </item>
               <item>
                <widget class="QPushButton" name="btn_keyctrl">
                 <property name="text">
                  <string>Keyboard control</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_7">
              <attribute name="title">
               <string>Drone commands</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="btn_takeoff">
                 <property name="text">
                  <string>Take off</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_land">
                 <property name="text">
                  <string>Land</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_goup">
                 <property name="text">
                  <string>Lift</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_godown">
                 <property name="text">
                  <string>Descendre</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_rotleft">
                 <property name="text">
                  <string>Rotate Left</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_rotright">
                 <property name="text">
                  <string>Rotate Right</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_chgCam">
                 <property name="text">
                  <string>Change camera</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_drone_disconnect">
          <property name="text">
           <string>Disconnect from the drone</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_drone_connect">
          <property name="text">
           <string>Connect to the Drone</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>frame_4</zorder>
       <zorder>frame</zorder>
       <zorder>label_3</zorder>
       <zorder>frame_3</zorder>
       <zorder>label_4</zorder>
       <zorder>label</zorder>
       <zorder>label_2</zorder>
       <zorder>frame_2</zorder>
       <zorder>btn_drone_disconnect</zorder>
       <zorder>btn_drone_connect</zorder>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Algorithms</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Learning</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="2">
            <widget class="QLineEdit" name="txt_path"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_creer">
             <property name="text">
              <string>Creer</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txt_path"/>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QPushButton" name="btn_load">
             <property name="text">
              <string>Charger les données</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btn_parcourir">
             <property name="text">
              <string>Parcourir</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Choix de l'Algorithme :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Données :</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBar_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTest2"/>
    <addaction name="actionTest4"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuBar_2"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Barre de status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <action name="actionTest2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTest4">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
